README.md

```diff
@@ -46,6 +46,9 @@ type Storage interface {
 	// Close closes the storage and will stop any running garbage
 	// collectors and open connections.
 	Close() error
+
+	// Returns the underlying DB client
+	Conn() db.Client
```

Comment:
I think we don't need here because of we dont add `Conn()` into the `Storage` interface.

Reply:
Done